js/press.js

```diff
@@ -106,15 +106,47 @@ function createCategoryTable(category, categoryLocation) {
             </div>`;
 }
 
+function createAerobaticEndTime(startTime, timeToAdd) {
+    let endTimeMinutes;
+    let endTimeHours;
+    let fixedStartTime = roundToMinute(startTime);
+    startTimeMinutes = Number(fixedStartTime.slice(3, 5) );
+    startTimeHours = Number(fixedStartTime.substr(0, 2));
+    tempMinutes = startTimeMinutes+timeToAdd;
+    if(tempMinutes >60) {
+        endTimeMinutes = tempMinutes-60;
+        endTimeHours = endTimeHours+1;
+    }
+    else {
+            endTimeMinutes = tempMinutes;
+            endTimeHours = startTimeHours;
+    }
+    let endTime = endTimeHours+':'+endTimeMinutes;
+    return endTime;
+}
+
 function createBaseTableTitle(name, activeTimes, hasAerobatic) {
-    let tempActiveTimes = activeTimes? `
+    let tempActiveTimesValue = activeTimes? activeTimes : null;
+    if(name.includes('מופע')) {
+        hasAerobatic = true;
+        let startTime = activeTimes.slice(0,5);
```

Comment:
```suggestion
        let startTime = activeTimes.slice(-5);
```
אם המחרוזת בפורמט `HH:mm - HH:mm` אז הכי קריא לקחת את סוף המחרוזת